url_shortener_proj/app/Exceptions/Handler.php

```diff
@@ -42,14 +47,20 @@ public function report(Throwable $exception)
     /**
      * Render an exception into an HTTP response.
      *
-     * @param  \Illuminate\Http\Request  $request
-     * @param  \Throwable  $exception
+     * @param \Illuminate\Http\Request $request
+     * @param \Throwable $exception
      * @return \Symfony\Component\HttpFoundation\Response
      *
      * @throws \Throwable
      */
     public function render($request, Throwable $exception)
     {
+        if ($this->isHttpException($exception)) {
+            if (view()->exists('errors.' . $exception->getStatusCode())) {
+                return response()->view('errors.' . $exception->getStatusCode(), [], $exception->getStatusCode());
+            }
+        }
```

Comment:
This functionality already exists in Laravel, you don't have to implement it by yourself. Take a look at this section of official documentation: https://laravel.com/docs/7.x/errors#http-exceptions